### ISSUE TITLE
[DOCUMENTATION]: Incorrect documentation

### ISSUE THREAD

**Issue by TeraShark** [dell/ansible-powerscale#27]
Samples in documentation are incorrect for the module.
All samples use the following:
```
dellemc.powerscale.filesystem:
  onefs_host: "{{powerscalehost}}"
  port: "{{powerscaleport}}"
  verify_ssl: "{{verify_ssl}}"
  username: "{{user}}"
  password: "{{password}}"
  ```
When using the samples as they are, errors are thrown, indicating that the inputs for the module are incorrect for "user, password and port".
However, according to the module parameters in the docs, the fields for User, Port and Password should be as follows:
```
dellemc.powerscale.filesystem:
  onefs_host: "{{ powerscalehost }}"
  port_no: "{{ powerscaleport }}"
  verify_ssl: "{{ verify_ssl }}"
  api_user: "{{ user }}"
  api_password: "{{ password }}"
  ```

**Comment by anupamaloke** [comment]
@TeraShark, thank you for bringing this to notice. We will work to get this fixed.

**Comment by anupamaloke** [comment]
Closed by #31
